docs/cookbooks/result.rst

```diff
@@ -637,6 +637,48 @@ The output appears as follows:
 
     Example Prefix $x^{\rm{g}} = 49.88^{+0.37}_{-0.35}$ & $norm^{\rm{g}} = 24.83^{+0.82}_{-0.76}$ & $\sigma^{\rm{g}} = 9.84^{+0.35}_{-0.40}$ \[-2pt]
 
+Derived Quantities (Advanced)

```

Comment:
Should this be called "Latent Samples"?

Reply:
Maybe, I am going to go over the API for this feature in the future.